### PR TITLE
bots: Add new triggers to image-refresh

### PR DESCRIPTION
**Pull request by marusak**
@martinpitt am I missing something or is something not correct?

### CODE REVIEW

File: bots/image-refresh [hunk 1]
```diff
@@ -74,7 +75,7 @@ TRIGGERS = {
         "ubuntu-stable"
     ],
     "openshift": [
-        # FIXME: need to test a rhel-7.x branch here, once we can
+        "fedora-30/container-kubernetes@cockpit-project/cockpit/rhel-7.7"
```
Comment (martinpitt) [inline bots/image-refresh]: trailing comma please, and let's run the "regular" rhel-7-7@cockpit-project/cockpit/rhel-7.7 tests as well, as they exercise kubernetes in a different way.

Comment (marusak) [reply bots/image-refresh]: Updated and triggered these new tests here to see, if they succeed or fail

**Comment by marusak** [comment]
I opened #12656 so we can trigger also the two remaining ones.

**Comment by martinpitt** [comment]
I landed #12656, tests-scan is happy about the two remaining tests now.

**Comment by marusak** [comment]
Cool that other branches worked :) So only kubernetes is broken. We should fix it in #12582

**Pull request merged**